### PR TITLE
Update view-compilation.md

### LINKED ISSUE #25350: docs should make note that implicit and global usings are NOT supported in runtime compilation
maybe a purple box or something?

cf https://github.com/dotnet/AspNetCore.Docs/issues/27390 for details on it not being implemented 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8a83deb2-e035-bb66-be4f-68709bcad341
* Version Independent ID: 16438f35-f35d-c097-0903-ec17e83e051e
* Content: [Razor file compilation in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/view-compilation?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/mvc/views/view-compilation.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/mvc/views/view-compilation.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-mvc**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #25350

### CODE REVIEW

File: aspnetcore/mvc/views/view-compilation.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Razor files with a `.cshtml` extension are compiled at both build and publish ti
 > [!NOTE]
 > Runtime compilation isn't supported for Razor components of Blazor apps.
 
+> [!NOTE]
+> [Global using directives](/dotnet/csharp/whats-new/csharp-10#global-using-directives) are ***not*** supported for runtime compilation.
```
Comment (Rick-Anderson) [inline aspnetcore/mvc/views/view-compilation.md:21]:
```suggestion
> [Global using directives](/dotnet/csharp/whats-new/csharp-10#global-using-directives) are ***not*** supported for runtime compilation.

```

**Comment by Rick-Anderson** [comment]
@adityamandaleeka please assign a reviewer.

**Comment by Rick-Anderson** [comment]
@adityamandaleeka please assign a reviewer. 30 second review.

**Comment by adityamandaleeka** [comment]
Sorry, missed this earlier. @mkArtakMSFT's team owns this part of the stack.

**Comment by Rick-Anderson** [comment]
@mkArtakMSFT please assign a reviewer.

**Comment by Rick-Anderson** [comment]
@mkArtakMSFT please assign a reviewer. 3 minute review

**Comment by Rick-Anderson** [comment]
> @mkArtakMSFT please assign a reviewer. 3 minute review

**Review by mkArtakMSFT — COMMENTED** [review]
Thanks @Rick-Anderson. Sorry for noticing this earlier.

In general, I am unable to keep up with GitHub notifications, given the number of issues I comment on. Hence, if something is urgent, feel free to message me via email or IM.

**Pull request merged**